packages/wmr/test/production.test.js

```diff
@@ -889,6 +889,40 @@ describe('production', () => {
 			const index = await fs.readFile(indexHtml, 'utf8');
 			expect(index).toMatch(/# hello world/);
 		});
+
+		// Whether or not to run the test to fetch external resources depends on whether or not a Node.js version 18 later
+		const isNodeVersionOver18 = Number(process.version.split('.')[0].slice(1)) >= 18;
+		// Even if you are using Node.js v18 or higher, you will get the error `globalThis.fetch is undefined` here, so get the version from process.version
+		// if (globalThis.fetch === undefined)
+		if (!isNodeVersionOver18) {
```

Comment:
nit: would get rid of the inversion here and switch the following code blocks or change the name.

`isNodeVersionUnder18` would probably be more clear.